### PR TITLE
Update CONTRIBUTING.md to list pull request and commit conventions

### PR DESCRIPTION
**Pull request by antiagainst**
These conventions will help us to speed up code review process
and keep a clean and healthy Git history for future reference.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
-(https://cla.developers.google.com/about/google-corporate).
+# Contributing to rspirv
+
+It's great to know that you are looking to contribute to rspirv! To keep the
+project in a consistent state on various aspects, we have the following
+conventions that it would be very nice if you can consider:
+
+## Code review
+
+* Please address comments with a new commit instead of force pushing to
```
Comment (kvark) [inline CONTRIBUTING.md]: TIL :)

**Comment by kvark** [comment]
Did you consider to move the `CONTRIBUTING.md` to `.github` folder (like in https://github.com/gfx-rs/gfx/tree/master/.github) so that the root dir is not polluted?

**Comment by antiagainst** [comment]
This is not specific to GitHub; so I think we should just keep it under root. Issue/pull request templates should be under `.github/`. :)

**Comment by kvark** [comment]
It does actually have github-specific treatment- https://help.github.com/en/articles/setting-guidelines-for-repository-contributors

**Pull request merged**